convert2rhel/toolopts.py

```diff
@@ -71,69 +71,68 @@ def _get_argparser():
     def _register_options(self):
         """Prescribe what command line options the tool accepts."""
         self._parser.add_option("-h", "--help", action="help", help="Show "
-                                " help message and exit.")
+                                                                    " help message and exit.")
```

Comment:
Why did you add this change here? Did you mean "    <text>" not             "<text>"?

Reply:
It`s a vscode format issue.. fixed.